src/connectors/sber-zvuk-dom-inject.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const getTrackInfo = (track, currentTime) => {
+	const { title, credits, duration, id, image, release_title: album } = track;
+	const trackArt = image.src.replace('{size}', '400x400');
+
+	return {
+		track: title,
+		artist: credits,
+		uniqueID: id,
+		duration,
+		trackArt,
+		currentTime,
+		album,
+	};
+};
+
+const queueEvent = (event) => {
+	let type = null;
+	let trackInfo; let queue;
+
+	if (event.type === 'update') {
```

Comment:
You can check for `event.type` first, and return earlier, if it's not equal "update". This allows to remove check if `type` (not event type) is not nullable.

Reply:
event.type is not nullable. `type` filled by mobx observer, and always has some type of action. We interested in `update` action, when some property have changed. I'll fix checking.

Reply:
> event.type is not nullable

I didn't say it's nullable. :)

My idea was to remove check for `type` value (which is from `mobx` observer). The `type` value is null, if we received an event with `event.type` != "update", so I suggested to check for `event.type` first, and return if it's not equal "update".

Reply:
already done :)